automate.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/sh
+
+#COLORS
+
+none='\033[0m'
+bold='\033[01m'
+disable='\033[02m'
+underline='\033[04m'
+reverse='\033[07m'
+strikethrough='\033[09m'
+invisible='\033[08m'
+
+black='\033[30m'
+red='\033[31m'
+green='\033[32m'
+orange='\033[33m'
+blue='\033[34m'
+purple='\033[35m'
+cyan='\033[36m'
+lightgrey='\033[37m'
+darkgrey='\033[90m'
+lightred='\033[91m'
+lightgreen='\033[92m'
+yellow='\033[93m'
+lightblue='\033[94m'
+pink='\033[95m'
+lightcyan='\033[96m'
+
+
+# Variables for folder names and paths 
+BUILD_F=build
+OUTPUT_F=build/output
+
+PATH_ONE=/root/libtorch
+PATH_TWO=/root//libtorch/share/cmake/Torch/
```

Comment:
Let's use more meaningful names, like:

```bash
PATH_LIBTORCH_DIR=/root/libtorch
PATH_LIBTORCH_BUILD=/root/libtorch/share/cmake/Torch/
```

Also, in `PATH_TWO`, was using `//` a mistake?

Reply:
yeah meaningful names are always better.... i did not know what the second path was actually pointing to....